### DISCUSSION TITLE
Can i get a alt link? My place has blocked the official site and some alts.

### DISCUSSION THREAD

**Discussion by NeboxArioNSYT** [QuiteAFancyEmerald/InvisiProxy#276] [General]
look at title.

**Comment by mistertoenails** [comment]
I have flexibleedu.gq, mutualbooks.biz, crownbuilding.design, and mathematica.live

**Reply by nonoguynono** [reply]
try not to post links on github, and send them on a private network, because school districts can easily block the links on here

**Reply by DevilGlitch** [reply]
intresting...
the website i used was blocked too

**Reply by MintvbzYT** [reply]
> 

My people blocked all yours.

**Reply by MintvbzYT** [reply]
My school mass blocks all the links shared publicly in GitHub.

**Reply by xxxfistoxxx** [reply]
can someone give me a link to roblox

**Reply by Marcelino319** [reply]
L

**Reply by Marcelino319** [reply]
none work for me

**Reply by FailerLover** [reply]
HOWWWWWWWW HOW HOW HOWWWW I AM NEWWWW

**Comment by Epic-gamer6969** [comment]
Copy this link and make your own website on replit.com so its easy to make new ones. ( like example.example.repl.co)

**Reply by SwissCheeseV** [reply]
Dont you need to buy it tho on replit

**Reply by idkidk738374** [reply]
as far as i know no

**Reply by DevilGlitch** [reply]
no, you just have to wake it up

**Comment by Gavyz** [comment]
Use replit.

**Reply by Prepple** [reply]
Can i have a link
And also how do I activate on replit

**Reply by Jeetboss** [reply]
Reality ain’t working it says that it gouges against their policy

**Comment by DevilGlitch** [comment]
also if you are able to get into the discord there is a link master bot
